static/src/javascripts/projects/commercial/modules/dfp/refresh-on-resize.ts

```diff
@@ -1,27 +1,37 @@
-import { hasCrossedBreakpoint, breakpoints } from '../../../../lib/detect';
+import { breakpoints, hasCrossedBreakpoint } from '../../../../lib/detect';
 import { mediator } from '../../../../lib/mediator';
-import { dfpEnv } from './dfp-env';
+import type { Advert } from './Advert';
 import { breakpointNameToAttribute } from './breakpoint-name-to-attribute';
+import { dfpEnv } from './dfp-env';
 import { refreshAdvert } from './load-advert';
 
-/* hasBreakpointChanged: ((string, string) -> undefined) -> undefined. Invokes the callback if a breakpoint has been crossed since last invocation */
+/**
+ * Invokes the callback if a breakpoint has been crossed since last invocation
+ */
 const hasBreakpointChanged = hasCrossedBreakpoint(true);
 
-/* breakpointNames: array<string>. List of breakpoint names */
-const breakpointNames = breakpoints.map((_) => _.name);
+/**
+ * Array of breakpoint names
+ */
+const breakpointNames = breakpoints.map(({ name }) => name);
 
 // TODO: reset advert flags
-const refresh = (currentBreakpoint, previousBreakpoint) => {
-	const getBreakpointIndex = (breakpoint, slotBreakpoints) => {
+const refresh = (currentBreakpoint: string, previousBreakpoint: string) => {
+	const getBreakpointIndex = (
+		breakpoint: string,
+		slotBreakpoints: string[],
+	) => {
 		const validBreakpointNames = breakpointNames
 			.slice(0, breakpointNames.indexOf(breakpoint) + 1)
 			.map(breakpointNameToAttribute);
 		return Math.max(
-			...slotBreakpoints.map((_) => validBreakpointNames.lastIndexOf(_)),
+			...slotBreakpoints.map((slotBreakpoint) =>
+				validBreakpointNames.lastIndexOf(slotBreakpoint),
+			),
```

Comment:
This is entirely your call as it's outside of remit of this PR as a TypeScript migration, but could we make this logic more readable?

I feel like it's tough to understand atm.

Reply:
I've actually started making some efforts to do this, but it looks like the mediator `on` listener that kicks off all this code never actually executes this callback, so this code doesn't run (on Frontend or DCR).

I'm inclined to merge this and figure out that issue separately (dead code with types is better than dead code without types?)